src/main/cpp/Subsystems/DriveTrain.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2014-2020 FRC Team 3512. All Rights Reserved.
+
+#include "Subsystems/DriveTrain.hpp"
+
+#include <frc/Talon.h>
+
+#include <cmath>
+
+DriveTrain::DriveTrain() {
+    m_deadband = 0.02f;
+    m_sensitivity = Constants::kLOW_GEAR_SENSITIVE;
+    // TODO Does robot start in low gear?
+
+    m_isDefencive = (false);
```

Comment:
This should be spelled `m_isDefensive`. Also, initialization with constants like this should be done in the header.